apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -245,6 +245,67 @@ function initializeBlocklyWrapper(blocklyInstance) {
     return blocklyWrapper.FieldTextInput;
   };
 
+  const oldMixin = blocklyWrapper.BlockSvg.prototype.mixin;
+  blocklyWrapper.BlockSvg.prototype.mixin = function(
+    mixinObj,
+    opt_disableCheck
+  ) {
+    oldMixin.call(this, mixinObj, true);
+  };
+
+  blocklyWrapper.BlockSvg.prototype.addUnusedBlockFrame = function(
+    helpClickFunc
+  ) {
+    if (!this.unusedSvg_) {
+      this.unusedSvg_ = new BlockSvgUnused(this, helpClickFunc);
+    }
+    this.unusedSvg_.render(this.svgGroup_);
+  };
+
+  const oldRender = blocklyWrapper.BlockSvg.prototype.render;
+  blocklyWrapper.BlockSvg.prototype.render = function(opt_bubble) {
+    oldRender.call(this, opt_bubble);
+    this.removeUnusedBlockFrame();
+  };
+
+  const oldDispose = blocklyWrapper.BlockSvg.prototype.dispose;
+  blocklyWrapper.BlockSvg.prototype.dispose = function() {
+    oldDispose.call(this);
+    this.removeUnusedBlockFrame();
+  };
+
+  blocklyWrapper.BlockSvg.prototype.isUnused = function() {
+    const isTopBlock = this.previousConnection === null;
+    const hasParentBlock = !!this.parentBlock_;
+    return !(isTopBlock || hasParentBlock);
+  };
+
+  blocklyWrapper.BlockSvg.prototype.removeUnusedBlockFrame = function() {
+    if (this.unusedSvg_) {
+      this.unusedSvg_.dispose();
+      this.unusedSvg_ = null;
+    }
+  };
+
+  blocklyWrapper.BlockSvg.prototype.getHexColour = function() {
+    // In cdo Blockly labs, getColour() returns a numerical hue value, while
+    // in newer Google Blockly it returns a hexademical color value string.
+    // This is only used for locationPicker blocks and can likely be deprecated
+    // once Sprite Lab is using Google Blockly.
+    return this.getColour();
+  };
+
+  blocklyWrapper.BlockSvg.prototype.isVisible = function() {
+    // TODO (eventually) - All Flappy/Poetry/Bounce blocks are visible.
```

Comment:
nit: we might want this to clarify that, "All Google Blockly apps' (currently, Flappy/Poetry/Bounce) blocks are visible"